OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -548,7 +548,7 @@ constant DebugFlag DUMP_ASSC = DEBUG_FLAG(189, "dumpASSC", false,
 constant DebugFlag SPLIT_CONSTANT_PARTS_SYMJAC = DEBUG_FLAG(190, "symJacConstantSplit", false,
   Gettext.gettext("Generates all symbolic Jacobians with splitted constant parts."));
 constant DebugFlag DUMP_FORCE_FMI_ATTRIBUTES = DEBUG_FLAG(191, "force-fmi-attributes", false,
-  Gettext.gettext("Dumps information from the optimization modules."));
+  Gettext.gettext("Dumps the fmi attributes with default values"));
```

Comment:
This might be misleading. I suggest something like 'Force to export all fmi attributes to the modelDescription.xml, including those which have default values.`.